### ISSUE TITLE
Example uses a color that is not described in any way in documentation.

### ISSUE THREAD

**Issue by ismagilli** [pgf-tikz/pgf#1159]
### Version

3.1.9a

### Details

Section 20.3 provides an example of a graph. To draw one of the lines, a color declared only in the documentation file:
https://github.com/pgf-tikz/pgf/blob/21310438ae418e15ea92b2845a866554202d13d7/doc/generic/pgf/pgfmanual-en-tikz-matrices.tex#L779
![2022-05-15_15-11-27](https://user-images.githubusercontent.com/38747776/168472115-aff0e905-3f0d-4529-9a85-ce126491b0f7.png)

It seems to me that this example should be corrected.

----

I have two a suggestion to improve this example.

First based on using `preaction`:
replace
```
\draw [-,line width=8pt,draw=graphicbackground]
      (D) to [bend right, looseness=1] (A);
\draw (D) to [bend right, looseness=1]
      node [near start] {b} node [near end] {e} (A);
```
to
```
\draw [preaction={-,line width=8pt,draw=graphicbackground}]
      (D) to [bend right, looseness=1]
      node [near start] {b} node [near end] {e} (A);
```

Second based on this [answer](https://tex.stackexchange.com/a/323315/171124): replace
```
\begin{scope}[every node/.style={font=\small\itshape}]
```
to
```
\begin{scope}[every node/.style={font=\small\itshape},
              transparency group=knockout]
```
and replace `draw=graphicbackground` to `draw,opacity=0`. This will make the outline drawing independent of the background.

**Comment by ismagilli** [comment]
I found another example in the documentation where this color is used, but it clearly states that this is the background color:
https://github.com/pgf-tikz/pgf/blob/21310438ae418e15ea92b2845a866554202d13d7/doc/generic/pgf/pgfmanual-en-tikz-graphs.tex#L69

**Comment by muzimuzhi** [comment]
I'm more in favor of adding similar comment to the first occurrence of `graphicbackground`.

Both examples use `graphicbackground` as the current background color, to manually implement some crossing/overlap effect. Not sure if an example for topic X should use advanced features in other topics or not. Using normal tricks may help newbies more focused on the current topic, while showing combinations of advanced features is a pursuit for perfection.

**Comment by Mo-Gul** [comment]
@a-is you are right, nowhere in the manual the definition of `graphicbackground` is given. But there are a lot of places in the manual where "information" is missing in examples. A few years ago even the needed libraries were missing. So that you get an impression, have a look at issue #640. I guess adding *all* needed information to *all* `codeexamples` would be an overkill. Therefore the idea was to have links to the individual `codeexamples` which then are fully defined and thus TeXable.

To support this you may have seen a few lines earlier to the quoted code line above that there _is_ stated the definition of the color.

https://github.com/pgf-tikz/pgf/blob/21310438ae418e15ea92b2845a866554202d13d7/doc/generic/pgf/pgfmanual-en-tikz-matrices.tex#L761

I agree that we could add a comment here as in the other example. Feel free to provide a PR.

@muzimuzhi, regarding the code _improvement_: For me this is ok for the first example, because in that particular case these are ...

https://github.com/pgf-tikz/pgf/blob/21310438ae418e15ea92b2845a866554202d13d7/doc/generic/pgf/pgfmanual-en-tikz-matrices.tex#L734-L737
